### ISSUE TITLE
Color contrast in links, header, footer

### ISSUE THREAD

**Issue by bengm** [krusynth/digitalpolicy#5]
The lighthouse report flags a few areas as insufficient color contrast. Report is attached, but it seems like it may be fixed by darkening:

1. Background of overall site title header
2. Background of overall site footer 
3. Hyperlink text color

[lighthouse-report.pdf](https://github.com/krusynth/digitalpolicy/files/8756098/lighthouse-report.pdf)

**Comment by krusynth** [comment]
Thanks for including the report - it looks like there are a few other things I could clean up here, but this should at least handle the contrast issue!
